doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -194,10 +199,23 @@ Modem libraries
 Event manager
 -------------
 
+* Removed:
```

Comment:
```suggestion
```
Should move to updated section as we are not completely removing the kconfig option.